mycpp/gc_heap.h

```diff
@@ -633,11 +637,11 @@ class Str : public gc_heap::Obj {
   bool isupper();
 
   Str* upper() {
-    assert(0);
+    NotImplemented(); // Uncalled
   }
 
   Str* lower() {
-    assert(0);
+    NotImplemented(); // Uncalled
```

Comment:
I think these must be called, if you 

`grep upper */*.py`

then it shows some usages. This is like `'mystr'.upper()` in Python

Reply:
If I change those function names to `upper_` and `lower_` I don't get compile or link errors.

The ones in `mylib/mycpp.h` line 337 and 341 break when I rename them.